src/System.Collections/src/System/Collections/Generic/Stack.cs

```diff
@@ -239,6 +241,12 @@ public T[] ToArray()
             return objArray;
         }
 
+        private void ThrowInvalidOperationEmptyStack()
```

Comment:
Nit: Any reason for the naming inconsistency between `ThrowForEmptyQueue` in #12062 and `ThrowInvalidOperationEmptyStack` here? It'd be nice if both methods were named consistently. I like the naming you have chosen here, so `ThrowInvalidOperationEmptyQueue` and `ThrowInvalidOperationEmptyStack`.